bakta/plot.py

```diff
@@ -334,7 +335,10 @@ def write_gc_content_skew(contigs, circos_path, colors):
     gc_skews = []
     max_gc = 0
     max_gc_skew = 0
-    gc_mean = SeqUtils.gc_fraction(''.join([c['sequence'] for c in contigs]))
+    if float(bp.__version__) >= 1.80:
+        gc_mean = SeqUtils.gc_fraction(''.join([c['sequence'] for c in contigs]))
+    else:
+        gc_mean = SeqUtils.GC(''.join([c['sequence'] for c in contigs])) / 100
```

Comment:
Is `/ 100` really required? It's not in the former code base: https://github.com/oschwengers/bakta/commit/52efd9d44f903cf487664b154fb369ded9366698. 

Cannot remember the details. Maybe w/o division by 100 was a lurking bug? Could you please double check & confirm this?